### ISSUE TITLE
Acoustic materials

### ISSUE THREAD

**Issue by jonlab** [jonlab/NewAtlantis#25]
As well as we have Materials (=graphics) and PhysicMaterials(=physics), we need "AcousticMaterials" to change the way that sound reflects (for example with the NAReflectionsFX component) on objects. This could be implemented as a component itself that would need to be applied to an object.
Parameters could be : Low frequency absorption, High frequency absorption, diffusion...

**Comment by rolandcahen** [comment]
Todo after LeCube performance if possible

**Comment by petesinc** [comment]
another (simple) approach would be % reflection ie one could increase reflection tp 99% if so desired.

**Comment by jonlab** [comment]
Yes, I think this is more consistent with the graphical aspects of 3D, how we handle colors and materials : with colors, a high value (white is RGB 1 1 1) means a high reflection, a low value (black 0 0 0) means a low reflection.

**Comment by jonlab** [comment]
As well as we have Reflection values, we can have Transmissions values, that control how the sound go through the object (that has this AcousticMaterial). All these AcousticMaterials would be used by the Audio pathfinding to construct the weighted graph.
